From fb764bdadc3aae23f7b2261120a1047ca92e69ba Mon Sep 17 00:00:00 2001
From: Benda Xu <heroxbd@gentoo.org>
Date: Thu, 11 Sep 2014 18:20:44 +0900
Subject: disable class and collaboration graph generation

Forwarded: no need
Description: The graphviz-generated class and collaboration graph
 occupies 1GB disk space.  It is too big to deal with.  We turn it
 off only to save space.
---
 doxygen.cfg | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doxygen.cfg b/doxygen.cfg
index 0fc2602..539cd73 100644
--- a/doxygen.cfg
+++ b/doxygen.cfg
@@ -34,8 +34,8 @@ PREDEFINED              = DOXYGEN_SHOULD_SKIP_THIS \
                           USE_THREADS \
 			  BOOST_PP_IS_ITERATING
 ALLEXTERNALS            = YES
-CLASS_GRAPH             = YES
-COLLABORATION_GRAPH     = YES
+CLASS_GRAPH             = NO
+COLLABORATION_GRAPH     = NO
 GROUP_GRAPHS            = NO
 GRAPHICAL_HIERARCHY     = NO
 DIRECTORY_GRAPH         = NO
